src/displayapp/screens/Notifications.h

```diff
@@ -57,6 +58,8 @@ namespace Pinetime {
           lv_obj_t* label_accept;
           lv_obj_t* label_mute;
           lv_obj_t* label_reject;
+          lv_obj_t *alert_type;
```

Comment:
I think it would be good to align the `*` to the type like the other lines do.

Reply:
Sure thing! I didn't notice... In this case I just do another commit and the pr automatically gets the last version?

Reply:
Yes, exactly.